linen_examples/pixelcnn/main.py

```diff
@@ -0,0 +1,74 @@
+# Copyright 2020 The Flax Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Main file for running the example.
+
+This file is intentionally kept short. The majority for logic is in libraries
+than can be easily tested and imported in Colab.
+"""
+
+from absl import app
+from absl import flags
+from absl import logging
+
+from clu import platform
+import sample
+import train
+import jax
+from ml_collections import config_flags
+import tensorflow as tf
+
+FLAGS = flags.FLAGS
+
+config_flags.DEFINE_config_file(
+    "config", "configs/default.py", "Training configuration.", lock_config=True)
```

Comment:
Done.
Every other popular programming language uses double quotes for strings. Who thought it would be a good idea to allow both in Python without providing a clear guideline?

Reply:
JavaScript also has this, BTW.